Dockerfile

```
@@ -0,0 +1,33 @@
+FROM golang:1.19.7-alpine
```

Maybe we should use always the latest version of Go `1.19`?
```suggestion
FROM golang:1.19-alpine
```

The benefit of this will be auto-applying security fixes.